rules/test-rule-jake/rule.md

```diff
@@ -0,0 +1,72 @@
+---
+type: rule
+title: Induction Task - Do you know this is a test rule
+uri: test-rule-jake
+authors:
+  - title: Jake Bayliss
+    url: https://www.ssw.com.au/people/jake-bayliss
+created: 2022-05-12T05:07:48.559Z
+guid: 73218c8b-b8d6-4d83-91f9-ae06cc9786ba
+---
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec scelerisque, metus eu iaculis suscipit, sem mauris gravida nunc, ac dictum nisl felis non nulla. Curabitur in orci nisi.
+
+<!--endintro-->
+
+# Rules for creating test rules
```

Comment:
I would avoid heading 1 inside the content... not a mistake, but h1 is normally used for the title

Reply:
Changed to heading 2